modules/arm_plugin/armplg_build.sh

```diff
@@ -29,14 +29,27 @@ fail()
     exit $retval
 }
 
+cloneSrcTree()
+{
+    DESTDIR=$1
+    shift
+    SRCURL=$1
+    shift
+    while [ "$#" -gt "0" ]; do
```

Comment:
Minor comment: do we need to convert number to string here?

Reply:
It's not necessary here. That's due to my habit to armor everything with parentheses, as a safer approach.
Fixed